app/api/helpers/payment.py

```diff
@@ -123,6 +120,13 @@ class PayPalPaymentsManager(object):
 
     @staticmethod
     def get_credentials(event=None, override_mode=False, is_testing=False):
+        """
+        Get Paypal credentials from settings module.
+        :param event: Associated event.
+        :param override_mode:
+        :param is_testing:
+        :return:
+        """
         if event and represents_int(event):
             event = safe_query(db, Event, 'id', event, 'event_id')
```

Comment:
Shouldn't this be the new safe_query for the non-deleted events, just to be sure?

Reply:
We don't need it @mayank8318 . See if someone tries to charge himself/herself on an order of a deleted event, they are welcome to do so. There is no need for us to check it everywhere in the project. Makes sense ?